Client/Transfers/factory.ts

```diff
@@ -66,13 +67,14 @@ export function getExpectedForBeneficiary(request: model.TransferRequest): model
 		amount: request.amount,
 		beneficiary: {
 			transferDestination: {
-				currency: request.currency,
+				currency: request.currency as Currency,
 				fullName: expect.any(String),
 			},
+			name: expect.any(String),
 			defaultReference: expect.any(String),
 			status: expect.stringMatching(/(ACTIVE)|(DELETED)|(OUTDATED)/),
```

Comment:
should this be a type instead?

Reply:
i should probably just delete this file

Reply:
but if we keep this i think it's fine as is and we could take a closer looks at tests another time